can_bus/json/PDM/PDM_alerts.json

```diff
@@ -11,7 +11,8 @@
         "StackWaterMarkHighTaskCanTx",
         "WatchdogTimeout",
         "TxOverflow",
-        "RxOverflow"
+        "RxOverflow",
+        "MissingBMSHeartbeat"
```

Comment:
The PDM now relies on the BMS for some signals -> You could make this a fault now